Shear/AppDelegate.swift

```diff
@@ -14,12 +14,13 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     @IBOutlet weak var window: NSWindow!
     
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/789b229ffaaa71902e329054d9d22803831cd48f/Shear/AppDelegate.swift#L15"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AppDelegate.swift:15</code></del></td>
    </tr>
  </tbody>
</table>